test/testdata/policy-controller/e2e/cip-keyless-with-attestations.yaml

```diff
@@ -33,8 +33,3 @@ spec:
         data: |
           predicateType: "cosign.sigstore.dev/attestation/v1"
           predicate: Data: "foobar e2e test"
```

Comment:
Why are you removing this?

Reply:
Is it intentional?

Reply:
Yeah, so with the move to OR, it's not really meaningful in our tests. So, in the tests now, we create first the keyless.yaml (or key.yaml) which check for signature, then the key[less]-with-attestations.yaml adds the AND by creating a new CIP so modified the tests to check for that as well. 
And cip-key-and-keyless.yaml tests the OR capability.
Otherwise, we'd have to bake the 'AND' (requires both signature and attestation) into the CIP policy, and I'd rather leave that to other PRs where we create better examples of CUE policies. I created #31 to track that issue.
@hectorj2f does that make sense?

Reply:
Yes, I understood it now. We could create more examples. Thanks